src/components/picker/api/pickerItem.api.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Picker.Item",
+  "category": "form",
+  "description": "Picker.Item, for configuring the Picker's selectable options",
+  "extends": ["TouchableOpacity"],
```

Comment:
It doesn't really extends TouchableOpacity, we don't pass on all TouchableOpacity props
```suggestion
```

Reply:
I'll remove from `PickerItem.js` as well